### ISSUE TITLE
Cannot install on Ubuntu 16.04

### ISSUE THREAD

**Issue by gabrik** [janestreet/core#108]
Hi all, I have issues on installing core on Ubuntu 16.04, Ocaml 4.06.0 opam 1.2.2

I have installed opam using 
` wget https://raw.github.com/ocaml/opam/master/shell/opam_installer.sh -O - | sh -s /usr/local/bin `

then 

    opam switch 4.06.0
    opam init -y
    opam install core -y

I got 

 
```
=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[ERROR] The compilation of ppx_fields_conv failed at "jbuilder build -p
        ppx_fields_conv -j 4".
[ERROR] The compilation of ppx_compare failed at "jbuilder build -p ppx_compare
        -j 4".
[ERROR] The compilation of ppx_sexp_conv failed at "jbuilder build -p
        ppx_sexp_conv -j 4".
[ERROR] The compilation of ppx_traverse failed at "jbuilder build -p
        ppx_traverse -j 4".

#=== ERROR while installing ppx_compare.v0.10.0 ===============================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ppx_compare -j 4
# path         /home/ubuntu/.opam/4.06.0/build/ppx_compare.v0.10.0
# compiler     4.06.0
# exit-code    1
# env-file     /home/ubuntu/.opam/4.06.0/build/ppx_compare.v0.10.0/ppx_compare-5898-6cc03f.env
# stdout-file  /home/ubuntu/.opam/4.06.0/build/ppx_compare.v0.10.0/ppx_compare-5898-6cc03f.out
# stderr-file  /home/ubuntu/.opam/4.06.0/build/ppx_compare.v0.10.0/ppx_compare-5898-6cc03f.err
### stderr ###
# File "/home/ubuntu/.opam/4.06.0/lib/ppx_inline_test/ppx_inline_test.dune", line 15, characters 0-0:
# Error: unclosed parentheses at end of input


#=== ERROR while installing ppx_fields_conv.v0.10.0 ===========================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ppx_fields_conv -j 4
# path         /home/ubuntu/.opam/4.06.0/build/ppx_fields_conv.v0.10.0
# compiler     4.06.0
# exit-code    1
# env-file     /home/ubuntu/.opam/4.06.0/build/ppx_fields_conv.v0.10.0/ppx_fields_conv-5898-c743ac.env
# stdout-file  /home/ubuntu/.opam/4.06.0/build/ppx_fields_conv.v0.10.0/ppx_fields_conv-5898-c743ac.out
# stderr-file  /home/ubuntu/.opam/4.06.0/build/ppx_fields_conv.v0.10.0/ppx_fields_conv-5898-c743ac.err
### stderr ###
# File "/home/ubuntu/.opam/4.06.0/lib/ppx_inline_test/ppx_inline_test.dune", line 15, characters 0-0:
# Error: unclosed parentheses at end of input


#=== ERROR while installing ppx_sexp_conv.v0.10.0 =============================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ppx_sexp_conv -j 4
# path         /home/ubuntu/.opam/4.06.0/build/ppx_sexp_conv.v0.10.0
# compiler     4.06.0
# exit-code    1
# env-file     /home/ubuntu/.opam/4.06.0/build/ppx_sexp_conv.v0.10.0/ppx_sexp_conv-5898-b76d46.env
# stdout-file  /home/ubuntu/.opam/4.06.0/build/ppx_sexp_conv.v0.10.0/ppx_sexp_conv-5898-b76d46.out
# stderr-file  /home/ubuntu/.opam/4.06.0/build/ppx_sexp_conv.v0.10.0/ppx_sexp_conv-5898-b76d46.err
### stderr ###
# File "/home/ubuntu/.opam/4.06.0/lib/ppx_inline_test/ppx_inline_test.dune", line 15, characters 0-0:
# Error: unclosed parentheses at end of input


#=== ERROR while installing ppx_traverse.v0.10.0 ==============================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ppx_traverse -j 4
# path         /home/ubuntu/.opam/4.06.0/build/ppx_traverse.v0.10.0
# compiler     4.06.0
# exit-code    1
# env-file     /home/ubuntu/.opam/4.06.0/build/ppx_traverse.v0.10.0/ppx_traverse-5898-8b8a2d.env
# stdout-file  /home/ubuntu/.opam/4.06.0/build/ppx_traverse.v0.10.0/ppx_traverse-5898-8b8a2d.out
# stderr-file  /home/ubuntu/.opam/4.06.0/build/ppx_traverse.v0.10.0/ppx_traverse-5898-8b8a2d.err
### stderr ###
# File "/home/ubuntu/.opam/4.06.0/lib/ppx_inline_test/ppx_inline_test.dune", line 15, characters 0-0:
# Error: unclosed parentheses at end of input
=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
```

I have tried also with ocaml 4.05.0 

Is this a bug or just my environment not working?

**Comment by diml** [comment]
What version of jbuilder are you using? There was an issue with 1.0+beta19, it's fixed in 1.0+beta19.1

**Comment by gabrik** [comment]
Thanks, I have the wrong version (1.0+beta19), I'll update to 1.0+beta19.1 

Thanks!
